app/models/provider.rb

```diff
@@ -234,7 +234,7 @@ def provider_type=(new_value)
   end
 
   def to_s
-    "#{provider_name} (#{provider_code})"
+    "[#{provider_code}] #{provider_name} (#{recruitment_cycle})"
```

Comment:
I know this has been merged, but I'd suggest the same as my comment here: https://github.com/DFE-Digital/manage-courses-backend/pull/749/files#r318904125

It's best to keep these representations consistent with the UI.